widgets/HTTP-streamingsource.json

```diff
@@ -133,6 +133,30 @@
             ]
           }
         },
+        {
+          "widget-type": "select",
+          "label": "Grant Type",
+          "name": "oauth2GrantType",
+          "widget-attributes": {
+            "values": [
+              "Refresh Token",
+              "Client Credentials"
+            ],
+            "default": "Refresh Token"
+          }
+        },
+        {
+          "widget-type": "select",
+          "label": "Client Authentication",
+          "name": "oauth2ClientAuthentication",
```

Comment:
This is just not evident from UI what do we mean by this property, can we please rename it to `Auth Credential Location`?

Or is there any standard name used by other products?

Please check with our product manager.

Reply:
similar comment for whole PR.

Reply:
Postman use Client Authentication, so we have used the same.

Reply:
Can you please confirm with our PM?

Reply:
This comment is still not addressed.

Reply:
We have confirmed that and he is fine with it.